src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/LocalDBTest/LocalDBTest.cs

```diff
@@ -90,6 +89,33 @@ public static void SqlLocalDbSharedInstanceConnectionTest()
         }
         #endregion
 
+        #region NamedPipeTests
+
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // No Registry support on UAP
+        [ConditionalFact(nameof(IsLocalDbNamedPipesEnabled))]
+        public static void SqlLocalDbNamedPipeConnectionTest()
+        {
+            ConnectionTest(s_localDbNamedPipeConnectionString);
```

Comment:
In these tests, you didn't do anything with the instance name while it's checked to be skipped by `IsLocalDbNamedPipesEnabled`!
Did you miss the test or just assess the wrong parameter?

Reply:
Test is using NamedPipe connection string, that is why the annotation is being used.